Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py

```diff
@@ -112,30 +114,37 @@ def get_data_with_mapped_label(data, labels_mapping, tag_field):
     return new_data, dict(exist_labels_counter), dict(missing_labels_counter)
 
 
-def store_model_in_demisto(model_name, model_override, X, y, confusion_matrix, threshold, y_test_true, y_test_pred,
-                           y_test_pred_prob):
-    model = demisto_ml.train_text_classifier(X, y, True)
-    model_data = demisto_ml.encode_model(model)
-    model_labels = demisto_ml.get_model_labels(model)
+def store_model_in_demisto(model_name, model_override, X, y, confusion_matrix, threshold, tokenizer, y_test_true,
+                           y_test_pred,
+                           y_test_pred_prob, target_accuracy):
+    PhishingModel = demisto_ml.PhishingModel(tokenizer)
```

Comment:
I think it's better to change variable name to"`phishing_model`, might lead to errors with the imports in future

Reply:
agree, name convention is lower case underscore